specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualNetworkGateway.json

```diff
@@ -1231,6 +1231,288 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/getFailoverAllTestsDetails": {
+      "post": {
+        "tags": [
+          "VirtualNetworkGateways"
+        ],
+        "operationId": "VirtualNetworkGateways_GetFailoverAllTestDetails",
+        "description": "This operation retrieves the details of all the failover tests performed on the gateway for different peering locations",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
```

Comment:
Why not using the common types?

Reply:
Keeping this consistent with all the other APIs in this file